lib/ES/Template.pm

```diff
@@ -73,6 +77,19 @@ my $Autosense_RE = qr{
         :(?:CONSOLE|AUTOSENSE|KIBANA):
     }xs;
 
+#===================================
+sub _build_js {
+#===================================
+    my ( $self ) = (@_);
+
+    if ($self->{compiled_js} == 0) {
+        print "Building docs.js\n";
+        run '/node_modules/parcel/bin/cli.js', 'build', '/build_docs/resources/web/docs_js/index.js', '-d', '/build_docs/resources/web', '-o', 'docs.js';
```

Comment:
Could we move this into `/tmp/` instead of writing it to the project directory? Most of the rest of the docs build uses `/tmp` for scratch work which is super safe because the docker container will vanish at the end of the run.

Reply:
Ah, well, everything expects `resources/web/docs.js` to exist, and this creates it.